class-gamajo-template-loader.php

```diff
@@ -110,6 +110,7 @@ public function __destruct() {
 		public function get_template_part( $slug, $name = null, $load = true ) {
 			// Execute code for this part.
 			do_action( 'get_template_part_' . $slug, $slug, $name );
+			do_action( 'get_template_part_' . $this->filter_prefix . '_' . $slug, $slug, $name );
```

Comment:
I think it should be:
```php
do_action( $this->filter_prefix . '_get_template_part_' . $slug, $slug, $name );
```
as that's how the `filter_prefix` property is used in two existing filters.